config/config.exs

```diff
@@ -11,6 +11,12 @@ if config_env() == :dev do
         ~w(js/app.js --bundle --minify --sourcemap=external --target=es2020 --outdir=../priv/static/),
       cd: Path.expand("../assets", __DIR__),
       env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+    ],
+    dev: [
```

Comment:
I'd change naming to e.g. `deploy_build` and `dev_build` to emphasize that it is executed by bot after merge